src/software/gui/generic_widgets/play_info/play_info_widget.cpp

```diff
@@ -2,17 +2,22 @@
 
 PlayInfoWidget::PlayInfoWidget(QWidget* parent) : QTextEdit(parent) {}
 
-void PlayInfoWidget::updatePlayInfo(const PlayInfo& play_info)
+void PlayInfoWidget::updatePlayInfo(const PlayInfoProto& play_info)
 {
     QString referee_command_string =
         QString("Referee Command: %1\n")
-            .arg(QString::fromStdString(play_info.getRefereeCommandName()));
+            .arg(QString::fromStdString(play_info.game_state().referee_command_name()));
     QString play_name_string =
-        QString("Play Name: %1\n").arg(QString::fromStdString(play_info.getPlayName()));
+        QString("Play Name: %1\n")
+            .arg(QString::fromStdString(play_info.play().play_name()));
     QString tactics_string = QString("Tactics:\n");
-    for (const auto& tactic_string : play_info.getRobotTacticAssignment())
+    for (const auto& tactic_string : play_info.robot_tactic_assignment())
```

Comment:
can we sort by robot id, so the list doesn't flicker?

Reply:
fixed